CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 * Removed unnecessary FindObjectOfType usage inside of ClientCharSelectState (#754)
 
 ### Fixed
-* Fixed up some minor art bugs (#762 #768)
+* Fixed up some minor art bugs (#762) (#768)
```

Comment:
This feels a little too vague. Shouldn't we outline what went into each item in the changelog?

Reply:
These particular changes are quite small, so I'm not sure they warrant having their own lines? We could do something like this:

* Resolved some minor art bugs (fixed clipping VFX, adjusted URP settings) (#762) (#768)

What do you think @SamuelBellomo? Since this was your suggestion after all :P

Reply:
That reads better, imo. As a user, it'd be tedious to go into the PR and skim through the changed files to find out what went in to the PR.

And little nitpick to introduce 768 details in its own PR.

Reply:
That's a good point! I'll split them up so that they're clearer